skrifa/src/color/mod.rs

```diff
@@ -76,6 +76,7 @@ pub enum PaintError {
     ParseError(ReadError),
     GlyphNotFound(GlyphId),
     PaintCycleDetected,
+    FillGlyphOptimizationFailed,
```

Comment:
This appears to only be used internally. Can we remove this and change back to a boolean flag for the optimization check?

Reply:
Changed to an internal boolean, removed enum value.